src/diff.ts

```diff
@@ -156,14 +159,20 @@ async function getBaseBuildForFeatureBranch(info: BuildkiteEnvironment): Promise
  */
 export async function getBaseBuild(info: BuildkiteEnvironment): Promise<BuildkiteBuild> {
   if (info.branch === info.integrationBranch) {
-    return getBaseBuildForIntegrationBranch(info, info.integrationBranch);
+    const build = await getBaseBuildForIntegrationBranch(info, info.integrationBranch);
+    log(`Found base build for integration branch: ${build.commit}`);
+    return build;
   }
 
   if (info.branch === info.defaultBranch) {
-    return getBaseBuildForDefaultBranch(info);
+    const build = await getBaseBuildForDefaultBranch(info);
+    log(`Found base build for default branch: ${build.commit}`);
+    return build;
   }
 
-  return getBaseBuildForFeatureBranch(info);
+  const build = await getBaseBuildForFeatureBranch(info);
+  log(`Found base build for feature branch: ${build.commit}`);
+  return build;
```

Comment:
These announce the final result, which we weren't actually doing (in non-integration branch cases, there was already something similar logged from `getSuitableBranchBuildAtOrBeforeCommit`, but these added logs will happen in every case, not just some cases)